esp8266/sonoffsc/sonoffsc.ino

```diff
@@ -19,7 +19,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include <Arduino.h>
-#include "config/all.h"
```

Comment:
Ah.  I am using visual studio (visual micro plugin) to edit my code, it does not deal with relative file references straight out the box so was not compiling.  I have found out how to add the files to the solution now. #653b4d3b4a7a0ea73c89bca997fa5b33bd28de5e